tests/unit/amo/components/TestPromotedAddonsCard.js

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+
+import AddonsCard from 'amo/components/AddonsCard';
+import PromotedAddonsCard, {
+  PromotedAddonsCardBase,
+} from 'amo/components/PromotedAddonsCard';
+import { createInternalAddon } from 'core/reducers/addons';
+import { fakeAddon, fakeI18n, shallowUntilTarget } from 'tests/unit/helpers';
+
+describe(__filename, () => {
+  const render = (customProps = {}) => {
+    const props = {
+      i18n: fakeI18n(),
+      loading: false,
+      ...customProps,
+    };
+
+    return shallowUntilTarget(
+      <PromotedAddonsCard {...props} />,
+      PromotedAddonsCardBase,
+    );
+  };
+
+  it('passes loading parameter to AddonsCard', () => {
+    const root = render({ loading: true });
+    expect(root.find(AddonsCard)).toHaveProp('loading', true);
+
+    root.setProps({ loading: false });
+    expect(root.find(AddonsCard)).toHaveProp('loading', false);
+  });
+
+  it('passes addons to AddonsCard', () => {
+    const addons = [
+      createInternalAddon({
+        ...fakeAddon,
+        slug: 'custom-addon',
+      }),
+    ];
+    const root = render({ addons });
+    expect(root.find(AddonsCard)).toHaveProp('addons', addons);
+  });
+
+  it('can pass a custom classname to AddonsCard', () => {
+    const className = 'some-class-name';
+    const root = render({ className });
+
+    expect(root.find(AddonsCard)).toHaveClassName(className);
+  });
+
+  it('passes addonInstallSource to AddonsCard', () => {
+    const addonInstallSource = 'featured-on-home-page';
+    const addons = [createInternalAddon(fakeAddon)];
+    const root = render({ addons, addonInstallSource });
+
+    expect(root.find(AddonsCard)).toHaveProp(
+      'addonInstallSource',
+      addonInstallSource,
+    );
+  });
+
+  it('uses the proper `rel` property for the header link in AddonsCard', () => {
+    const root = render();
+
+    expect(root.find(AddonsCard).html()).toContain('rel="noopener noreferrer"');
```

Comment:
I feel like there has to be a better way than this, but I couldn't figure out how to drill down into the `header` prop and have it be an enzyme object.

Reply:
In the past, I think we retrieved the `header` prop and then used `shallow()` like here: https://github.com/mozilla/addons-frontend/blob/4dec24c1a8666c75f54f08f9c3e4a7d35d09c68d/tests/unit/amo/pages/TestCollection.js#L717-L718

Reply:
I tried that and it didn't work. It said I couldn't apply `shallow` to it.  I'll try again.

Reply:
I had to wrap the prop in a `<div>` and then it would allow me to apply `shallow` to it.